cli.go

```diff
@@ -768,6 +786,12 @@ Options:
       Set the maximum staleness and allow stale queries to Consul which will
       distribute work among all servers instead of just the leader
 
+  -no-prefix[=<bool>]
+	  Tells Envconsul to not prefix the keys with their parent "folder".
+	  Default for -prefix is true, default for -secret is false.
+	  Differing defaults is to maintain backward compatibility.
+	  Leaving off the bool value is the same as setting to true.
```

Comment:
I think we can document this flag as `-no-prefix` instead of `-no-prefix[=<bool>]` like we do with `-once` and `-sanitize`. That's simpler and then we don't need this last line either.

Reply:
I mentioned the argument for -no-prefix as, unlike the other cases, both forms are useful as each of the 2 prefix systems controlled (the consul keys and vault's secrets) have opposite defaults.

Reply:
I do think that is a bit wordy for cli reference output. It should only contain what it does. I think adding the other information to the documented config file in the Readme would be a better place for it.